docs/src/submodules/Utilities/overview.md

```diff
@@ -424,34 +547,16 @@ function _copy_to(dest::Optimizer, src::Model)
         row_bounds.upper,
     )
     # Set objective sense and constant (omitted)
-    return
-end
-
-function MOI.copy_to(dest::Optimizer, src::Model)
```

Comment:
This is an important bit we need to keep. The whole point of this is that once `default_cache` returns an `OptimizerCache`, JuMP will use this cache so 99.99% of the time, `copy_to` will be called with an `OptimizerCache` as second argument. `copy_to` with an abstract `MOI.ModelLike` is only there for the rare use-case of some using Clp with MOI and not instantiating it with `MOI.instantiate` (then the cost of the copy is solely in building the index map, see https://github.com/jump-dev/MathOptInterface.jl/issues/1772).
If the only `copy_to` method implemented is the one taking `MOI.ModelLike`, it completely defeats the purpose.

Reply:
Ah I forgot about `default_cache`. Updated with comments.